### PR TITLE
Add WinSCP support for Linux hosts

### PR DESCRIPTION
**Pull request by sturner88**
Open WinSCP for linux hosts from context menu

### CODE REVIEW

File: QuickConnectPlugin/QuickConnectPluginExt.cs [hunk 1]
```diff
@@ -193,6 +194,44 @@ public class QuickConnectPluginExt : Plugin {
...
+                            cmd.StartInfo.UseShellExecute = false;
+                            cmd.Start();
+                            cmd.StandardInput.WriteLine(String.Format("\"{0}\" scp://{2}:{3}@{1}",
+                                winScpPath,
+                                hostPwEntry.IPAddress,
+                                hostPwEntry.GetUsername(),
+                                hostPwEntry.GetPassword())
+                                );
+                            cmd.StandardInput.Flush();
+                            cmd.StandardInput.Close();
```
Comment (cristianst85) [inline QuickConnectPlugin/QuickConnectPluginExt.cs]: There are a few thing that need to be done here:
- create a new class `WinSCPArgumentsFormatter` implementing `IArgumentsFormatter`
- add a test case for `WinSCPArgumentsFormatter`
- refactor using `ProcessUtils.StartDetached()`

**Comment by sturner88** [comment]
I have implemented the changes. Take a look and let me know if it's ok.

**Comment by cristianst85** [comment]
Looks good to me.

Now we have a non-coding issue, regarding the [copyright of the WinSCP icon](https://winscp.net/eng/docs/license#license_of_winscp_icon_set):

> Note that all images distributed in or with WinSCP application are NOT published under the GPL license. It means that it is not allowed to redistribute or reuse these images or parts of them or modifications of them without WinSCP separately or in or with another software.

I see three possible solutions:
- to ask for special permission to use/distribute the icon (highly improbable);
- find an alternative icon with a compatible license (LGPL/GPL, Creative Commons);
- or use no icon (for now at least).

P.S. This is also the reason that I've not used the original VMWare icons, etc.

**Comment by cristianst85** [comment]
Thanks for your contribution!

**Pull request merged**